functions/memorystore/redis/function.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_memorystore_redis]
+
+package visitcount
+
+import (
+  "fmt"
+  "net/http"
+  "os"
+
+  "github.com/gomodule/redigo/redis"
+)
+
+var redisPool *redis.Pool
+
+// Initialize the connection pool on instance startup
+func init() {
+  redisHost := os.Getenv("REDISHOST")
+  redisPort := os.Getenv("REDISPORT")
```

Comment:
Please check if either of these variables is empty and `log.Fatal` if so.

```go
if redisHost == "" {
  log.Fatal("REDISHOST must be set")
}
redisPort := os.Getenv("REDISPORT")
if redisPort == "" {
  log.Fatal("REDISPORT must be set")
}
```

Also, are these standard names for these variables? Wondering if they should be `REDIS_HOST` and `REDIS_PORT`.

Reply:
Done, and I'm not sure if they're standard but our other Memorystore samples use these names. Figured I'd be consistent.